### PR TITLE
fix: format boolean values as true/false

### PR DESCRIPTION
**Pull request by magnew**
Closes: #6101

### Description:

Users were confused that boolean values in results were formatted as Yes/No. This changes booleans in tables and filters to be 'True' or 'False'. 

Before:
Screenshot 2023-08-23 at 14 51 27

After:
Screenshot 2023-08-23 at 14 50 32

### CODE REVIEW

File: packages/common/src/utils/formatting.ts [hunk 1]
```diff
@@ -58,7 +58,9 @@ export const currencies = [
 ];
 
 export const formatBoolean = <T>(v: T) =>
-    ['True', 'true', 'yes', 'Yes', '1', 'T'].includes(`${v}`) ? 'Yes' : 'No';
+    ['True', 'true', 'yes', 'Yes', '1', 'T'].includes(`${v}`)
```
Comment (joaoviana) [inline packages/common/src/utils/formatting.ts:61]: I believe you need to update tests in: `/packages/common/src/utils/formatting.test.ts`

Not sure if we need to update docs but didn't really see a reference to `yes` on our codebase!

**Review by joaoviana — CHANGES_REQUESTED** [review]
one check to make!

File: packages/common/src/utils/formatting.ts (hunk 1 above)
Comment (magnew) [reply packages/common/src/utils/formatting.ts:61]: Thanks, good call!

**Review by joaoviana — APPROVED** [review]
Nice! Once tests (and e2e) pass, let's merge :)

**Pull request merged**